src/components/TooltipNew/SpotlightOverlay.js

```diff
@@ -33,6 +33,14 @@ const SpotlightOverlay = ({ gap, isVisible, slide }) => {
     height: `calc(100% - ${gap.bottom}px)`
   };
 
+  const spotlight = {
+    top: `${gap.top}px`,
+    left: `${gap.left}px`,
+    width: `${gap.width}px`,
+    height: `${gap.height}px`,
+    opacity: 0
```

Comment:
since `visibility` is still `visible` it will take over all click event right?

Reply:
ok